### PR TITLE
Basic Structure of Coach Reports' Summary Section

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary
Added **basic** structure of  coach reports' summary section.

## Screenshot
image

### CODE REVIEW

File: kolibri/plugins/coach_tools/assets/src/actions.js [hunk 1]
```diff
@@ -1,17 +1,75 @@
...
+  // Check if params are valid.
+  if (!(CONTENT_SCOPE_OPTIONS.includes(contentScope)
+    && USER_SCOPE_OPTIONS.includes(userScope)
+    && ALL_OR_RECENT_OPTIONS.includes(allOrRecent)
+    && VIEW_BY_CONTENT_OR_LEARNERS_OPTIONS.includes(viewByContentOrLearners))) {
+    // If incorrect redirect to default reports.
+    alert('Invalid query!');
+    redirectToDefaultReports(store, params);
+  } else {
+    console.log('Valid query.');
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/assets/src/actions.js:65]: We can probably use the `handleError(store, errorString)` action from `core-actions` here to give a user-facing error in addition to a console error.

File: kolibri/plugins/coach_tools/assets/src/app.js [hunk 2]
```diff
@@ -14,15 +14,24 @@ class CoachToolsModule extends KolibriModule {
...
       }
     );
 
     router.on(
-      PageNames.SCRATCHPAD,
-      '/scratchpad',
+      PageNames.REPORTS_NO_QUERY,
+      '/reports',
       (toRoute, fromRoute) => {
-        actions.showScratchpad(store);
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/assets/src/app.js:25]: let's keep a copy of the scratchpad around - it comes in handy for prototyping new components

File: kolibri/plugins/coach_tools/assets/src/actions.js (hunk 1 above)
Comment (christianmemije) [reply kolibri/plugins/coach_tools/assets/src/actions.js:65]: Okay, and we should still redirect to the default reports?

File: kolibri/plugins/coach_tools/assets/src/app.js (hunk 2 above)
Comment (christianmemije) [reply kolibri/plugins/coach_tools/assets/src/app.js:25]: Oops, I kind of replaced it with reports. I can push a scratchpad if you want.

File: kolibri/plugins/coach_tools/assets/src/actions.js (hunk 1 above)
Comment (indirectlylit) [reply kolibri/plugins/coach_tools/assets/src/actions.js:65]: Probably not necessary at this point - the default error handling is very loud and we need better handling of it globally, not just here.

**Pull request merged**